tests/test_symbolicate.py

```diff
@@ -331,42 +331,12 @@ def test_client_happy_path_v5(json_poster, clear_redis_store, botomock, metricsm
     assert response["Access-Control-Allow-Origin"] == "*"
     assert "content-type" in response["Access-Control-Allow-Headers"]
 
-    metrics_records = metricsmock.get_records()
-    metricsmock.print_records()
-    assert metrics_records[0] == (
-        INCR,
-        "tecken.symbolicate_symbolication",
-        1,
-        ["version:v5"],
+    metricsmock.assert_incr("tecken.symbolicate_symbolication", tags=["version:v5"])
+    metricsmock.assert_incr(
+        "tecken.symbolicate_symbolication_jobs", tags=["version:v5"]
     )
-    assert metrics_records[1] == (
-        INCR,
-        "tecken.symbolicate_symbolication_jobs",
-        1,
-        ["version:v5"],
-    )
-    assert metrics_records[2] == (
-        INCR,
-        "tecken.symbolicate_symbol_key",
-        1,
-        ["cache:miss"],
-    )
-    assert metrics_records[3] == (
-        INCR,
-        "tecken.symbolicate_symbol_key",
-        1,
-        ["cache:miss"],
-    )
-
-    # The reason these numbers are hardcoded is because we know
-    # predictable that the size of the pickled symbol map strings.
-    metricsmock.has_record(GAUGE, "tecken.storing_symbol", 76)
-    metricsmock.has_record(GAUGE, "tecken.storing_symbol", 165)
-
-    # Called the first time it had to do a symbol store
-    metricsmock.has_record(GAUGE, "tecken.store_keys", 1, None)
-    # Called twice because this test depends on downloading two symbols
-    metricsmock.has_record(GAUGE, "tecken.store_keys", 2, None)
```

Comment:
These aren't actually asserting anything because `.has_record()` returns a bool. The code that these were testing was removed, so these tests are now vestigial.